sbt-plugin/src/test/scala/org/scalajs/sbtplugin/ScalaJSConfigJSONProtocolTest.scala

```diff
@@ -0,0 +1,94 @@
+package org.scalajs.sbtplugin
+
+import org.junit.Test
+import org.junit.Assert._
+
+import ScalaJSConfigJSONProtocol._
+import org.scalajs.linker.interface._
+
+import sjsonnew.JsonFormat
+import sjsonnew.BasicJsonProtocol._
+import sjsonnew.support.scalajson.unsafe.Converter.{fromJsonUnsafe, toJsonUnsafe}
+
+class ScalaJSConfigJSONProtocolTest {
```

Comment:
Is this the best place to put this test? How can I add it to the CI?

Reply:
Add `sbtPlugin/test` here: https://github.com/scala-js/scala-js/blob/master/Jenkinsfile#L364